### PR TITLE
Unify text for installer

### PR DESCRIPTION
**Pull request by wli3**
windows-1
mac
deb

#6448

### CODE REVIEW

File: packaging/osx/clisdk/resources/cs.lproj/firstTimeWelcomeMessage.html [hunk 1]
```diff
@@ -7,22 +7,22 @@
 <body>
     <font face="Helvetica">
         <h3>
-            Welcome to .NET Core!
+            Installation Successfully Completed Welcome to .NET Core!
```
Comment (livarcocc) [inline packaging/osx/clisdk/resources/cs.lproj/firstTimeWelcomeMessage.html]: Installation Completed Successfully?

File: packaging/osx/clisdk/resources/de.lproj/firstTimeWelcomeMessage.html [hunk 2]
```diff
@@ -7,22 +7,22 @@
 <body>
     <font face="Helvetica">
         <h3>
-            Welcome to .NET Core!
+            Installation Successfully Completed Welcome to .NET Core!
```
Comment (livarcocc) [inline packaging/osx/clisdk/resources/de.lproj/firstTimeWelcomeMessage.html]: Same here.

File: packaging/osx/clisdk/resources/en.lproj/firstTimeWelcomeMessage.html [hunk 3]
```diff
@@ -7,22 +7,22 @@
 <body>
     <font face="Helvetica">
         <h3>
-            Welcome to .NET Core!
+            Installation Successfully Completed Welcome to .NET Core!
```
Comment (livarcocc) [inline packaging/osx/clisdk/resources/en.lproj/firstTimeWelcomeMessage.html]: Same here.

Comment (livarcocc) [inline packaging/osx/clisdk/resources/en.lproj/firstTimeWelcomeMessage.html]: Well, you get the gist.

**Comment by wli3** [comment]
@richlander could you review the user facing text?

File: packaging/osx/clisdk/resources/cs.lproj/firstTimeWelcomeMessage.html (hunk 1 above)
Comment (wli3) [inline packaging/osx/clisdk/resources/cs.lproj/firstTimeWelcomeMessage.html]: Changed to Installation Completed Successfully

File: packaging/osx/clisdk/resources/cs.lproj/firstTimeWelcomeMessage.html [hunk 4]
```diff
@@ -7,22 +7,22 @@
 <body>
     <font face="Helvetica">
         <h3>
-            Welcome to .NET Core!
+            Installation Completed Successfully Welcome to .NET Core!
```
Comment (nguerrera) [inline packaging/osx/clisdk/resources/cs.lproj/firstTimeWelcomeMessage.html]: Neither "completed" nor "successfully" should be capitalized. There also needs to be some punctuation between the two parts to this.

The following edit would address my feedback, but please get the text reviewed doc writers.
```
Installation completed successfully. Welcome to .NET Core!
```

File: packaging/osx/clisdk/resources/cs.lproj/firstTimeWelcomeMessage.html [hunk 5]
```diff
@@ -7,22 +7,22 @@
...
         </h3>
         <p>
             Learn more about .NET Core at <a href="https://aka.ms/dotnet-docs">https://aka.ms/dotnet-docs</a> . Use dotnet --help to see available commands or go to <a href="https://aka.ms/dotnet-cli-docs">https://aka.ms/dotnet-cli-docs</a>.
         </p>
         <h3>
-            Telemetry
+            NET Core Tools Telemetry
```
Comment (nguerrera) [inline packaging/osx/clisdk/resources/cs.lproj/firstTimeWelcomeMessage.html]: This needs the leading dot. Product name is ".NET Core"

**Comment by richlander** [comment]
Feel free to merge. I will likely come back with some additional changes but don't want to block progress.

File: packaging/osx/clisdk/resources/cs.lproj/firstTimeWelcomeMessage.html (hunk 4 above)
Comment (wli3) [inline packaging/osx/clisdk/resources/cs.lproj/firstTimeWelcomeMessage.html]: Fixed at 303634978aaaa9a13f6c97cef2c62e1a73519e5a

File: packaging/osx/clisdk/resources/cs.lproj/firstTimeWelcomeMessage.html (hunk 5 above)
Comment (wli3) [inline packaging/osx/clisdk/resources/cs.lproj/firstTimeWelcomeMessage.html]: Fixed at 3813c419481bc6abcd154598cf0e7d015436fdd1

**Comment by wli3** [comment]
@dotnet-bot test Windows_NT x64 Release Build

**Comment by wli3** [comment]
@MattGertz Could you review this change?

**Comment by wli3** [comment]
Unify text for installer: deb pkg exe installer's telemetry notice and welcome message

**Pull request merged**